### DISCUSSION TITLE
Runelite failing to launch, states OSRS servers are temporarily offline. Servers are not offline.

### DISCUSSION THREAD

**Discussion by Tapedstuff** [runelite/runelite#16352] [Q&A] [answered]
Runelite is failing to launch throwing an error stating the OSRS servers are temporarily offline, but the servers aren't offline. I've tried restarting my laptop, changing from my wifi to a mobile hotspot, uninstalling the Jagex launcher and Runelite, then redownloading and reinstalling both. The issue is persisting. I've attached a screenshot of the launcher error message. Meanwhile, OSRS mobile is working just fine on my phone. I've tried searching Reddit, the Q&A, and the troubleshooting FAQ and have had no luck so far.

![runeliteissue](https://user-images.githubusercontent.com/127177802/223331045-29b23bb7-c0f3-44b1-ac53-919734bd8306.jpg)

**Comment by Nightfirecat** [comment, marked as answer]
This message is set in advance of [an upcoming outage between 10am and 11:30am GMT](https://secure.runescape.com/m=news/game-status-information-centre?oldschool=1). Please join our [Discord server](https://runelite.net/discord) for client support.
